### PR TITLE
Project4: CSS3 Flexbox & Grid

### PR DESCRIPTION
**Pull request by 3rkeen**
This project is completed by using CSS flexbox and grid. It has a fixed header, main body for article and footer for links. Footer includes one fixed division to show whether the user has logged in or not.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,931 @@
...
+          Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team,
+          knocked on his door with a bottle of Champagne to tell him the news.
+        </p>
+        <iframe
+          width="600"
+          height="319"
+          src="https://www.youtube.com/embed/ZlfIVEy_YOA"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
```
Comment (euqueme) [inline index.html]: - [ ] move the width and height from the iframe to a class in the CSS file since no inline styling is allowed in the HTML file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @3rkeen and @Jmagero :balloon: :lollipop: 

### Good job so far, the page looks very similar to the original and your CSS file it's optimal :ok_hand: there is still some work to be done :construction: :construction_worker_woman: :construction_worker_man:  please address the comments I've made in your code and next :point_down: 

## Page structure and Implementation details

#### Navigation bar

- [ ] It should have a fixed position and all the elements (menu, search icon, space and cosmos link, new york times logo, and buttons) should be links or have a pointer when hovered

image

#### The theory of inflation

- [ ] you are missing the [theory of inflation background image](https://static01.nyt.com/newsgraphics/2014/03/18/theory-of-inflation/1ebaa1b972edda0f407fbaa0e010f6b50fb6792e/inflation-945.png) let's add it 

image

#### Bottom part

* [ ] It contains links (`<a>` tags that show a pointer when hovered) to More in Space and Astronomy and Editor's pics with image previews on the leftmost side. 
* [ ] It contains links (`<a>` tags that show a pointer when hovered) to most popular articles on the rightmost side. 
* [ ] you have More in Space and Astronomy title repeated 2 times the bottom one should be change to Editor's picks

#### Footer

- [ ] Add the footer bar that the page has when the user is not logged in and this should have a fixed position 

image

## General

#### Descriptive pull request

- [ ] Edit your pull request to explain more about what you did in the project, one line is not enough let's add a couple more

#### In line styling

- [ ] I found inline styling in your code so please address the comment I've left bellow :arrow_double_down: 

### That's it, team! :raised_hands: 

### Please :pray: do the changes :woman_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Comment by 3rkeen** [comment]
Dear  Maria,

Please check the updated version of the project where all the errors has
been corrected.

Thank you.

**Comment by uryelah** [comment]
Hello **Jocyline Magero** & **Erkin Sheraliev**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**Good job fixing the changes from the last cr 🎇 🎆 **
There's still some work to be done so that your project is good to go, so hang in there.

---------    

**Great job at: 🥇** 
- positioning the main text of the page
- using semantic HTML
- using icons at the start of the article
- paying attention to text color

**You still need to fix the issues below before you can move to the next project**

---------

### Required changes 🛠 

- [ ] Like the last TSE pointed out, the **More in space and Astronomy** and **Editor Picks** recommended articles are all clickable, have the pointer cursor when you hover over them, and not only the text, **the images are also clickable**. 
[Check the original site for reference.](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)

- [ ] Good job adding the 'Theory of Inflation' Images but take some time to fix the font-weight also. Compare your version and the original one bellow.
Screenshot_2020-03-06 Document
Screenshot_2020-03-06 Space Ripples Reveal Big Bang’s Smoking Gun(1)

- [ ] Please add icons to the Subscribe area of the footer. Compare your version and the original one bellow:

Screenshot_2020-03-06 Document(1)
Screenshot_2020-03-06 Space Ripples Reveal Big Bang’s Smoking Gun(2)

- [ ] Fix your **Most Popular** links so they have the same font size and margin as the original ones.
Screenshot_2020-03-06 Document(2)
Screenshot_2020-03-06 Space Ripples Reveal Big Bang’s Smoking Gun(3)


---------
### Optional Enhancements

- [ ] [OPTIONAL] The [original site](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) has a lot of nice hover effects. Check especially how the links change their style when you hover your mouse over them.
 How about trying to add them too? It improves the user experience greatlyCheck this for reference on :hover.
---------
 
**Please fix the required changes 👷  before asking for a new code review**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by 3rkeen** [comment]
Dear Sarah,

We have fixed all the issues you mentioned. Please check.

Thank you.

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [ ] 'Editor's Pick' is overlapping with the main text, consider to fix this as the original page:



Original page:



- [ ] Social icons should be a link, consider check the original page social icons to see that they are links to click.

[OPTIONAL]

- [ ] Pull Request has a meaningful title and a short description, [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/
) you can learn about it. 

- [ ] Consider adding the new version on your 'demo link' in your readme file.

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by majovanilla** [comment]
Great work team!! :star: 

You made all the required changes and you are able to merge.

Status: **approved**

**Pull request merged**